server/graphql/v2/mutation/ExpenseMutations.ts

```diff
@@ -528,7 +533,9 @@ const expenseMutations = {
         status: expenseStatus.DRAFT,
       });
 
-      await sendDraftExpenseInvite(req, expense, collective, draftKey);
+      if (!args.skipInvite) {
+        await sendDraftExpenseInvite(req, expense, collective, draftKey);
+      }
```

Comment:
One issue s that `sendDraftExpenseInvite` also takes care of recording the activity, which is important for the activity log / audit trail. I'd suggest setting `activity.data.notify` to false, see https://github.com/opencollective/opencollective-api/blob/ff571d0b0ae40fca506387b4e507119e40d2ab80/server/models/Activity.ts#L109. Also make sure there's no email reminder CRON?